src/components/landing-pages/image-section-item.tsx

```diff
@@ -1,6 +1,6 @@
 import { Flex, FlexProps } from "@chakra-ui/react";
 import { StaticImageData } from "next/image";
-import React from "react";
+import * as React from "react";
```

Comment:
Can we use named imports here instead?

Reply:
even better we don't need this at all here - that said in the `/ui` stuff from shadcn its always there, wonder if it's a config